src/Tizen.TV.UIControls.Forms/Renderer/GridViewRenderer.cs

```diff
@@ -0,0 +1,168 @@
+        {
+            if (Control == null)
+            {
+                _genGrid = new ElmSharp.GenGrid(Xamarin.Forms.Forms.NativeParent)
+                {
+                    HorizontalScrollBarVisiblePolicy = Element.IsHorizontalScrollBarVisible == false ? ElmSharp.ScrollBarVisiblePolicy.Invisible : ScrollBarVisiblePolicy.Visible,
+                    VerticalScrollBarVisiblePolicy = Element.IsHorizontalScrollBarVisible == false ? ElmSharp.ScrollBarVisiblePolicy.Invisible : ScrollBarVisiblePolicy.Visible,
+                    IsHorizontal = Element.IsHorizontal,
+                    ItemWidth = Element.ItemWidth,
+                    ItemHeight = Element.ItemHeight,
+                    ItemAlignmentX = Element.ItemHorizontalAlignment,
+                    ItemAlignmentY = Element.ItemVerticalAlignment,
+                    Style = Element.ThemeStyle,
+                };
+                _genGrid.ItemSelected += OnItemSelected;
+                _genGrid.ItemFocused += OnItemFocused;
+                _genGrid.ItemUnfocused += OnItemUnfocused;
+                
+                if (Element.ItemsSource != null)
+                {
+                    UpdateItemsSource();
+                }
+                SetNativeControl(_genGrid);
+            }
+            base.OnElementChanged(e);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == GridView.ItemWidthProperty.PropertyName)
+            {
+                Control.ItemWidth = Element.ItemWidth;
+            }
+            else if (e.PropertyName == GridView.ItemHeightProperty.PropertyName)
+            {
+                Control.ItemHeight = Element.ItemHeight;
+            }
+            else if (e.PropertyName == GridView.ThemeStyleProperty.PropertyName)
+            {
+                Control.Style = Element.ThemeStyle;
+            }
+            else if (e.PropertyName == GridView.ItemsSourceProperty.PropertyName)
+            {
+                UpdateItemsSource();
+            }
+            base.OnElementPropertyChanged(sender, e);
+        }
+
+        void OnItemSelected(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SelectedItem = context.Data;
+        }
+
+        void OnItemUnfocused(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SendItemFocused(new GridViewFocusedEventArgs(context.Data, context.RealizedView, false));
+        }
+
+        void OnItemFocused(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SendItemFocused(new GridViewFocusedEventArgs(context.Data, context.RealizedView, true));

```

Comment:
why don't use IXXXController pattern?

e.g.) 
```cs

(Element as IGridViewController )?.SendItemFocused(new GridViewFocusedEventArgs(context.Data, context.RealizedView, true));


public interface IGridViewController : IViewController
{
...
}

public class GridView : View, IGridViewController 
{
...
}
```

Reply:
I will change to use IViewController. Thank you.